pkg/kubectl/resource_printer_test.go

```diff
@@ -1103,3 +1103,45 @@ func TestPrintPodWithLabels(t *testing.T) {
 		buf.Reset()
 	}
 }
+
+func TestPrintPodWithWideFormat(t *testing.T) {
```

Comment:
Optional: this would probably be easier to read as an Example instead of a test. Just replace `buf` with `os.Stdout`, and put the expected output in an `Output:` comment. See [ExamplePrintReplicationController](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/cmd_test.go#L244), which does a very similar thing. Bonus points if you want to move that test here .